src/main/drivers/io.c

```diff
@@ -194,10 +185,8 @@ void IOHi(IO_t io)
 {
     if (!io)
         return;
-#if defined(USE_LOWLEVEL_DRIVER)
+#if defined(USE_HAL_DRIVER)
```

Comment:
No, it's technically still HAL but using the LL. the low level driver flag was only intermediary whilst mixing it up for a given driver for a given target i.e. SPI but only for Nero 

Now I'll just mix and match whole drivers, e.g. convert i2c for all targets, so no need to separate HAL from LL